src/SumoCoders/FrameworkCoreBundle/Tests/Controller/DatePickerControllerTest.php

```diff
@@ -148,44 +100,26 @@ public function testIfSingleTextWithDatePickerAndOnlyDatesBetweenCorrectly()
         $endDate = new \DateTime('next friday');
 
         $element = $crawler->filter('input#form_date_example8');
-        $wrapper = $element->parents()->filter('.date-widget')->first();
 
         $this->assertEquals(1, $element->count());
-        $this->assertEquals(1, $wrapper->count());
 
         // check if it has all the required data-attributes
-        $this->assertEquals('datepicker', $wrapper->attr('data-provider'));
-        $this->assertEquals($date->format('Y-m-d'), $wrapper->attr('data-date'));
-        $this->assertEquals('form_date_example8', $wrapper->attr('data-link-field'));
-        $this->assertEquals('yyyy-mm-dd', $wrapper->attr('data-link-format'));
-        $this->assertEquals('range', $wrapper->attr('data-date-type'));
-        $this->assertEquals($startDate->format('Y-m-d'), $wrapper->attr('data-minimum-date'));
-        $this->assertEquals($endDate->format('Y-m-d'), $wrapper->attr('data-maximum-date'));
+        $this->assertEquals($startDate->format('d/m/Y'), $element->attr('data-date-start-date'));
+        $this->assertEquals($endDate->format('d/m/Y'), $element->attr('data-date-end-date'));
 
         // check if the actual element is hidden
-        $this->assertEquals('hidden', $element->attr('type'));
-        $this->assertEquals($date->format('Y-m-d'), $element->attr('value'));
+        $this->assertEquals($date->format('d/m/Y'), $element->attr('value'));
     }
 
     public function testIfSingleTextWithoutDefaultDateRenderedCorrectly()
     {
         $crawler = $this->getCrawlerForRequest('GET', '/_tests/datepicker');
 
         $element = $crawler->filter('input#form_date_example9');
```

Comment:
don't forget to adjust this id when shifting the ids in the code

Reply:
fixed 89a0a2e